### PR TITLE
SymPy Mechanics for Autolev Users guide

### PR DESCRIPTION
**Pull request by NikhilPappu**
- other
  -  Added a guide pydy_for_autolev_users.rst in the mechanics part of the documentation.

### CODE REVIEW

File: doc/src/modules/physics/mechanics/autolev_parser.rst [hunk 1]
```diff
@@ -147,9 +147,453 @@ statements demonstrate how to get responses similar to the ones in the
...
+  1. Inertia declarations (``INERTIA B,I1,I2,I3``) set the inertias of rigid
+  bodies.
+  
+  2. Inertia setters of the form ``I_C_D>> = expr`` however, set the inertias
+  only when C is a body. If C is a particle then ``I_C_D>> = expr``
+  simply parses to ``i_c_d = expr`` and ``i_c_d`` acts like a regular variable.
+  
+  3. When it comes to inertia getters (``I_C_D>>`` used in an expression or
+  ``INERTIA`` commands), these MUST be used with the ``EXPRESS`` command
+  to specifiy the frame as SymPy needs this information to compute the
```
Comment (Abdullahjavednesar) [inline doc/src/modules/physics/mechanics/autolev_parser.rst]: specifiy -> specify

File: doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst [hunk 2]
```diff
@@ -0,0 +1,732 @@
...
+| input code. Its code is a lot     | their code for let's say a type of|
+| cleaner and concise as a result of| rigibodies with common            |
+| this.                             | properties.                       |
+|                                   | The wide array of scientific      |
+|                                   | Python libraries available is also|
+|                                   | a big plus.                       |
++-----------------------------------+-----------------------------------+
+||                                  ||                                  |
+| Autolev generates Matlab, C, or   | PyDy generates numerical Python,  |
+| Fortan code from a small set of   | C or Octave/Matlab code from a    |
```
Comment (Abdullahjavednesar) [inline doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst]: Fortan -> Fortran

File: doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst [hunk 3]
```diff
@@ -0,0 +1,732 @@
...
+
+Links
+----------
+
+`SymPy Tutorial <http://docs.sympy.org/latest/tutorial/index.html>`_
+
+`SymPy Documentation <http://docs.sympy.org/>`_
+
+`SymPy Physics Vector
+Dcumentation <http://docs.sympy.org/latest/modules/physics/vector/index.html>`_
```
Comment (Abdullahjavednesar) [inline doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst]: Dcumentation -> Documentation

**Comment by NikhilPappu** [comment]
Sorry about the small commit names. I have just merged stuff from other branches or tried to make changes so the Travis tests pass.

**Comment by asmeurer** [comment]
If this document is about PyDy specifically shouldn't it go in the PyDy documentation rather than the SymPy documentation?

**Comment by asmeurer** [comment]
CC @moorepants

**Comment by NikhilPappu** [comment]
Aaron,
We have decided that it would be best to have this in SymPy as the parser
is here. The guide mostly talks about SymPy vs Autolev code and just
touches on PyDy. I have used the name "PyDy" in the title as it is broader
and encompasses the SymPy mechanics module. I could change the name if
required.

**Comment by moorepants** [comment]
@asmeurer I think we can just rename it something like "SymPy Mechanics for Autolev Users". It belongs here for now. The vast majority of the code it is referencing is in SymPy.

File: doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst [hunk 4]
```diff
@@ -0,0 +1,732 @@
+.. _pydy_for_autolev_users:
+
+======================
+PyDy for Autolev Users
```
Comment (moorepants) [inline doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst]: Change to ``SymPy Mechanics for Autolev Users``.

File: doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst [hunk 5]
```diff
@@ -0,0 +1,732 @@
...
+Autolev (now superseded by MotionGenesis) is a domain specific programming
+language which is used for symbolic multibody dynamics. The SymPy mechanics
+module now has enough power and functionality to be a fully featured symbolic 
+dynamics module. The PyDy package extends the SymPy output to the numerical
+domain for simulation, analyses and visualization. Autolev and PyDy have
+a lot in common but there are also many differences between them.
+This page shall expand upon their differences. It is meant to be a go-to
+reference for Autolev users who want to transition to PyDy.
+
+It would be nice to have a basic understanding of SymPy and PyDy before
```
Comment (moorepants) [inline doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst]: Orient references to PyDy to "SymPy Mechanics".

File: doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst [hunk 6]
```diff
@@ -0,0 +1,732 @@
...
+|                       | ``(fr, frstar) =      |                       |
+|                       | KM.kanes_equations(fL,|                       |
+|                       | bL)`` where fL and bL |                       |
+|                       | are lists of forces   |                       |
+|                       | and bodies            |                       |
+|                       | respectively.         |                       |
++-----------------------+-----------------------+-----------------------+
+
+Numerical Evaluation and Visualization
+----------------------------------------
```
Comment (moorepants) [inline doc/src/modules/physics/mechanics/pydy_for_autolev_users.rst]: You can mention PyDy (instead of SymPy Mechanics) in this section.

**Review by moorepants — CHANGES_REQUESTED** [review]
This looks graat. Please address the comments on changing the language to reflect "SymPy Mechanics" instead of PyDy for everything above the numerical code section and this can be merged.

**Review by moorepants — APPROVED** [review]

**Pull request merged**